doc/api/fs.md

```diff
@@ -2481,7 +2481,7 @@ changes:
   * `stats` {fs.Stats}
 
 Retrieves the {fs.Stats} for the symbolic link referred to by the path.
-The callback gets two arguments `(err, stats)` where `stats` is a {`fs.Stats}
+The callback gets two arguments `(err, stats)` where `stats` is a {`fs.Stats`}
```

Comment:
We seem to be stylizing "{...} object" without backticks? cc @nodejs/documentation

Reply:
Yes, removing the backticks makes it a link, which would be preferable IMHO.
```suggestion
The callback gets two arguments `(err, stats)` where `stats` is a {fs.Stats}
```